lib/galaxy/jobs/runners/slurm.py

```diff
@@ -48,7 +48,7 @@ class SlurmJobRunner(DRMAAJobRunner):
 
     def _complete_terminal_job(self, ajs, drmaa_state, **kwargs):
         def _get_slurm_state_with_sacct(job_id, cluster):
-            cmd = ['sacct', '-n', '-o', 'state%-32']
+            cmd = ['sacct', '-n', '-o', 'state%-32,exit']
```

Comment:
We should probably use the [official field names](https://slurm.schedmd.com/sacct.html), i.e. `State%-32,ExitCode`